### PR TITLE
MSVC fix -> add 'dealii::' root namespace explicitely where MSVC fail…

### PR DESCRIPTION
**Pull request by l-korous**
…s to understand ' internal::' without 'dealii::'

### CODE REVIEW

File: tests/trilinos/precondition_muelu_dgp.with_trilinos>=11.14.with_64bit_indices=off.output [hunk 1]
```diff
@@ -1,3 +0,0 @@
-
-DEAL:12288::Solver stopped within 18 - 32 iterations
-DEAL:49152::Solver stopped within 18 - 32 iterations
```
Comment (bangerth) [inline tests/trilinos/precondition_muelu_dgp.with_trilinos>=11.14.with_64bit_indices=off.output]: same issue with these files.

File: include/deal.II/base/tensor.h [hunk 2]
```diff
@@ -892,7 +892,11 @@ inline
 typename Tensor<rank_,dim,Number>::value_type &
 Tensor<rank_,dim,Number>::operator[] (const unsigned int i)
 {
-  return internal::TensorSubscriptor::subscript(values, i, internal::int2type<dim>());
+  // A fix for MSVC (prepending the 'dealii::')
+  // MSVC > C:\hpfem\agros2d\dealii\include\deal.II/base/tensor.h(886): error C2872: 'internal' : ambiguous symbol
+  // - could be 'dealii::internal'
+  // - or 'dealii::`anonymous-namespace'::internal'
+  return dealii::internal::TensorSubscriptor::subscript(values, i, dealii::internal::int2type<dim>());
```
Comment (bangerth) [inline include/deal.II/base/tensor.h]: just drop the extra comment. This sort of information can be gotten from the git annotations, but in reality nobody will ever wonder why we need `dealii::` in some places.

File: include/deal.II/base/tensor.h [hunk 3]
```diff
@@ -901,7 +905,8 @@ inline
 const typename Tensor<rank_,dim,Number>::value_type &
 Tensor<rank_,dim,Number>::operator[] (const unsigned int i) const
 {
-  return internal::TensorSubscriptor::subscript(values, i, internal::int2type<dim>());
+  // See a note in the body of Tensor<rank_,dim,Number>::operator[] (const unsigned int i)
```
Comment (bangerth) [inline include/deal.II/base/tensor.h]: same here -- just drop the comment.

**Comment by l-korous** [comment]
Done.

**Pull request merged**